src/probnum/randprocs/_random_process.py

```diff
@@ -155,6 +160,28 @@ def cov(self, args0: _InputType, args1: Optional[_InputType] = None) -> _OutputT
         """  # pylint: disable=trailing-whitespace
         raise NotImplementedError
 
+    def covmatrix(
```

Comment:
Could you add a docstring to `cov`? Similar to the one of the `var` and `std` methods?